src/web/app/pages-instructor/instructor-course-details-page/instructor-course-details-page.component.ts

```diff
@@ -195,7 +187,7 @@ export class InstructorCourseDetailsPageComponent implements OnInit {
   deleteAllStudentsFromCourse(courseId: string): void {
     this.studentService.deleteAllStudentsFromCourse({ courseId })
       .subscribe((resp: MessageOutput) => {
-        this.loadCourseDetails(courseId);
+        this.loadStudents(this.courseDetails.course.courseId);
```

Comment:
If we delete all student, we can actually just clear the list (by setting to `[]`)